test/system/700-play.bats

```diff
@@ -30,7 +30,7 @@ spec:
   containers:
   - command:
     - sleep
-    - "100"
+    - \"100\"
```

Comment:
This and line 52 are completely unrelated; it's just something that makes color highlighting work better in my editors.